src/Services/Tenor/TenorDataReader.cs

```diff
@@ -0,0 +1,198 @@
+                    AddressHouseLetter = f.Bostedsadresse.FirstErGjeldende()?.Vegadresse.Adressekode,
+                    AddressHouseNumber = f.Bostedsadresse.FirstErGjeldende()?.Vegadresse.Adressenummer.Husnummer,
+                    AddressMunicipalName = null,
+                    AddressPostalCode = null,
+                    AddressStreetName = null,
+                    Email = null,
+                    MobileNumber = null,
+                    TelephoneNumber = null,
+                    Language = null,
+                    MailingAddress = null,
+                    MailingPostalCity = null,
+                    MailingPostalCode = null,
+                    PartyRoles = fnrRoleLookup.TryGetValue(fnr, out var partyRoles) ? partyRoles : new Dictionary<int, List<Role>>(),
+                    CustomClaims = new()
+                    {
+                        new()
+                        {
+                            Type = "user:source",
+                            ValueType = "http://www.w3.org/2001/XMLSchema#string",
+                            Value = "localTenor"
+                        }
+                    },
+
+                };
+            }).ToList(),
+            Orgs = brreg.Select(b => new AppTestOrg
+            {
+                PartyId = b.PartyId,
+                ParentPartyId = null,
+                OrgNumber = b.Organisasjonsnummer,
+                Name = b.Navn,
+                BusinessAddress = string.Join("\n", b.Forretningsadresse.Adresse),
+                BusinessPostalCity = b.Forretningsadresse.Poststed,
+                BusinessPostalCode = b.Forretningsadresse.Postnummer,
+                MailingAddress = string.Join("\n", b.Postadresse.Adresse),
+                MailingPostalCity = b.Postadresse.Poststed,
+                MailingPostalCode = b.Postadresse.Postnummer,
+                EMailAddress = null,
+                FaxNumber = null,
+                InternetAddress = null,
+                MobileNumber = null,
+                TelephoneNumber = null,
+                UnitStatus = null,
+                UnitType = null,
+            }).ToList(),
+        };
+    }
+
+    public async Task StoreUploadedFile(IFormFile file)
+    {
+        var dir = GetTenorStorageDirectory();
+        var filename = new FileInfo(Path.Join(dir.FullName, file.FileName));
+        if (filename.Directory?.FullName != dir.FullName)
+        {
+            throw new Exception($"Invalid filename {file.FileName}");
+        }
+        using Stream fileStream = filename.OpenWrite();
+        await file.CopyToAsync(fileStream);
+    }
+    private static T? ParseCatchException<T>(string rawJson) where T : class
+    {
+        try
+        {
+            return JsonSerializer.Deserialize<T>(rawJson, _options);
+        }
+        catch (Exception)
+        {
+            return null;
+        }
+    }
+    public async Task<List<TenorFileItem>> GetFileItems()
```

Comment:
Started localtest in docker and when trying to open UserAdministration site I get an exception `DirectoryNotFoundException: Could not find a part of the path '/AltinnPlatformLocal/tenorUsers'.`

Reply:
Ooops, I seem to have added another listing before using. Now it gets explicitly created.